quinn/src/connection.rs

```diff
@@ -106,6 +106,16 @@ impl Future for Connecting {
     }
 }
 
+impl Connecting {
+    /// Get the peer address of a `Connecting`.
+    ///
+    /// This must not be called after `poll` has returned `Ready`, or it will panic.
+    pub fn peer_address(&self) -> SocketAddr {
```

Comment:
`Connection` calls this `remote_address`. I'm not specifically attached to that, but we should be consistent (and probably word their documentation the same, too).